config/helm/chart/default/values.yaml

```diff
@@ -52,6 +52,4 @@ csidriver:
 
 securityContextConstraints:
   enabled: true # Only applicable for Openshift
-  
-activeGate:
```

Comment:
is this not used anymore?

Reply:
yep, it used to set a feature flag on the CR